transformers/events/DOCUMENTATION.md

```diff
@@ -27,7 +27,7 @@ If the contract isn't already present in the environment you'll need to add it b
    
     ``` toml
     [contract]
-    [contrac t.MCD_FLIP_ETH_A]
+    [contrac t.MCD_FLIP_ETH_A_1.0.0]
```

Comment:
Why is there a space between c and t here?

Reply:
yikes, there shouldn't be!

Reply:
At least it's just docs.